engines/twine/debug.grid.cpp

```diff
@@ -0,0 +1,126 @@
+/* ScummVM - Graphic Adventure Engine
```

Comment:
I'm not comfortable with having a filename with multiple dots in it. Whilst I don't think there's any specific rule against it, it's more standard practice to use an underscore, like "debug_grid.cpp"

Reply:
I've renamed them